ethcore/src/state/mod.rs

```diff
@@ -395,6 +395,13 @@ impl<B: Backend> StateInfo for State<B> {
 const SEC_TRIE_DB_UNWRAP_STR: &'static str = "A state can only be created with valid root. Creating a SecTrieDB with a valid root will not fail. \
 			 Therefore creating a SecTrieDB with this state's root will not fail.";
 
+/// Constructs the MKVS key used to store account metadata.
+pub fn get_metadata_key(address: &[u8]) -> Vec<u8> {
```

Comment:
```suggestion
pub fn get_metadata_key(address: Vec<u8>) -> Vec<u8> {
```
unless for API convenience, it's generally better to take an owned thing if you want an owned thing and let the caller determine how best to obtain it